tools/cloud-build/daily-tests/ansible_playbooks/test-htcondor-access-point.yml

```diff
@@ -13,10 +13,17 @@
 # limitations under the License.
 
 ---
-- name: Ensure schedd has joined pool
+- name: Ensure schedd has joined the pool
   ansible.builtin.command: condor_status -schedd -autof Name
   register: schedd_status
   changed_when: False
-  until: ansible_fqdn == schedd_status.stdout
+  until: schedd_status.stdout == ansible_fqdn
   retries: 8
   delay: 15
+- name: Ensure execute points have joined the pool
+  ansible.builtin.command: condor_status -collector -autoformat HostsTotal
+  register: hosts_total
+  changed_when: False
+  until: hosts_total.stdout == "2"
```

Comment:
IIUC this magic number is coming from `community/examples/htc-htcondor.yaml` L122. Can we do more to connect the dots here. One idea would be turn this into a variable which would at least allow you to name it and default to 2. The name could be something like `htcondor_execute_point_min_idle_nodes`. In absence of that at least a comment.